src/models/User.ts

```diff
@@ -47,7 +47,7 @@ const UserModel = {
       WIDGETS.FEATUREDSHORTCUTS,
       WIDGETS.GUARDIANIDEAL,
     ]
-
+    console.log('New user to Create 🔥 in MongoDB: ', newUser)
```

Comment:
I'd change this to `info` or `debug` unless we don't need it.

Reply:
these are just my debugging logs, will for sure change or remove :)